src/phoneField/PhoneField.tsx

```diff
@@ -256,11 +255,11 @@ export class PhoneField extends PureComponent<PhoneFieldProps, PhoneFieldState>
 
     return (
       <div className={cc([baseClassName, prefix({ error: !!error }), this.props.className])}>
-        <div id={id} className={classNames} aria-labelledby={ariaLabelledBy}>
+        <div id={id} className={classNames} {...a11yAttrs}>
```

Comment:
`id={id}` should also be generated by `{...a11yAttrs}`
I'm not sure it's a good idea to have `id` handled by A11yProps but since it's there already.

Reply:
For the logic of the ID, it's mainly because a lot of aria attributes are also used alongside an `id`: `aria-labelledby`, `aria-describedby`, etc... all point to an `id`. So I've added it to the list so we can have those behaviors working without  having to add extra props.

Removed the specific call to `id` there since it's handled by `a11yAttrs`.